qabelbox/src/main/java/de/qabel/qabelbox/fragments/BaseFragment.kt

```diff
@@ -15,7 +17,10 @@ import org.jetbrains.anko.longToast
 import org.jetbrains.anko.onUiThread
 
 
-abstract class BaseFragment : Fragment(), QblView {
+abstract class BaseFragment(protected val mainFragment: Boolean = false,
+                            protected val showOptionsMenu: Boolean = false,
+                            protected val showFABButton: Boolean = false) : Fragment(), QblView {
```

Comment:
just curious ... FAButton ... FloatingActionButton ... FAB ... but why FABButton? It's like LCDDisplay 😆